File: Examples/DotNET/SDK/VisualBasic/Document/RejectTrackingChanges.vb
Imports Aspose.Cloud

Namespace Document
    Class RejectTrackingChanges
        Public Shared Sub Run()
            Dim inputfile As String = "doc-sample.doc"

            ' Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, String.Empty)

            ' Reject All Tracking Changes in the Document
            Dim wordRevisionResponse2 As WordRevisionResponse = Common.WordsService.WordsRevisions.RejectAllRevisionsInDocument(inputfile, Common.GetOutputFilePath(inputfile, True), Common.FOLDER)

            Console.WriteLine(Convert.ToString(vbLf & "All tracking changes rejected in ") & inputfile)
        End Sub
    End Class
End Namespace

File: Examples/DotNET/SDK/CSharp/Document/RejectTrackingChanges.cs
using System;
using Aspose.Cloud;

namespace CSharp.Document
{
    class RejectTrackingChanges
    {
        public static void Run()
        {
            string inputfile = "doc-sample.doc";

            // Upload input file from local directory to Cloud Storage
            Common.UploadFile(inputfile, string.Empty);

            // Reject All Tracking Changes in the Document
            WordRevisionResponse wordRevisionResponse2 = Common.WordsService.WordsRevisions.RejectAllRevisionsInDocument(inputfile, Common.GetOutputFilePath(inputfile, true), Common.FOLDER);

            Console.WriteLine("\nAll tracking changes rejected in " + inputfile);
        }
    }
}
